actionpack/lib/action_dispatch/routing.rb

```diff
@@ -243,9 +243,9 @@ module ActionDispatch
   #
   #   rails routes
   #
-  # Target specific controllers by prefixing the command with <tt>-c</tt> option. Use
-  # <tt>--expanded</tt> to turn on the expanded table formatting mode.
-  #
+  # Target specific controller by prefixing the command with <tt>-c</tt> option,
+  # or grep routes using <tt>-g</tt> option. Use <tt>--expanded</tt> to turn on
+  # the expanded table formatting mode.
```

Comment:
`the expanded table formatting mode` doesn't really explain anything to users. How about:

```
# Useful in conjunction with <tt>--expanded</tt> which displays routes vertically.
```